content/posts/2019-05-21---Working-In-Ember.md

```diff
@@ -0,0 +1,174 @@
+</figure>
+
+I'm not going to claim that I'm the best Ember developer in the world but since you may not know who I am, I figured I'd at least give _some_ reasons why my opinion isn't completely without merit.
+I worked at a startup that wrote Ember and Elixir for a couple of years.
+During that time, we contracted with [DockYard](http://dockyard.com), who was pretty well known for Ember in the world and especially so in Boston.
+When that startup didn't last (because that's what happens at startups, unfortunately...), I actually went to DockYard.
+My time at DockYard was mostly spent in Ember (with some Rails sprinkled in for good measure), where I spent most of my time (about 2 years) contracted to work for the biggest content provider in the world.
+
+During my time with Ember, I've [contributed to the framework itself](https://github.com/emberjs/data/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aclosed+author%3Asbatson5+), written [multiple blog posts](https://dockyard.com/blog/authors/scott-batson) and helped [author](https://github.com/sbatson5/ember-wistia) and [maintain](https://github.com/poteto/ember-cli-flash) many open source projects.
+I also have a [YouTube Channel](https://www.youtube.com/c/scottbatson) where I've done [a lot of Ember tutorials](https://www.youtube.com/playlist?list=PLelL5s7nDFKVVPZNwfnuDG-Xe_28F1PSO) with about 65,000 views.
+My [Ember Data tutorial](https://www.youtube.com/watch?v=ljLxZw-XStw) has over 16,000 views at the moment (which I recorded long before I considered myself "senior" by any means).
+
+I'm not trying to brag (...ok, maybe a little) but I just wanted to say that I have been a part of the Ember.js Community for quite a while and as more than just a passive user.
+So, hopefully my opinion isn't discredited immediately.
+
+# What's the point of this post?
+
+This isn't meant to serve as a "this is why Ember is great" or "this is why Ember is bad" post (although I certainly have opinions on both).
+It's just meant to be a reflection on the framework, community, the greater web community and maybe just a bit of personal reflection.
+
+![Ember Trophies](/media/ember.png)
+
+## Ember is not dead and everyone knows it
+
+<figure>
+	<blockquote>
+		<p>Lol, people think Ember is dead but look at these random stats!</p>
+    <footer>
+			<cite>— Overly defensive web devs.</cite>
+		</footer>
+	</blockquote>
+</figure>
+
+I feel like I see a ton of tweets and posts from really smart people in the community answering the question "is Ember dead?" with a resounding "heck no!"
+But honestly, I don't actually see a lot of people asking that question (except for the occasional troll).
+Nobody thinks Ember is going to disappear -- there are a lot of companies that are invested in it, a community that is passionate about maintaining it and enough smart people behind it to help drive it into the future.
+Nobody seriously thinks it is going to just end and disappear, so we need to stop being overly defensive about this.
```

Comment:
A little repetitive with the "nobody thinks ... disappear" twice but that may be intentional.

Totally agree with you on this point, by the way. It's kind of the "me thinks thou doth protest too much" type of thing.